docs/Examples/Chunked/Main.purs

```diff
@@ -0,0 +1,39 @@
+module Examples.Chunked.Main where
+
+import Prelude
+
+import Effect.Console as Console
+import HTTPure as HTTPure
+import Node.Stream as Stream
+
+-- | Serve the example server on this port
+port :: Int
+port = 8091
+
+-- | Shortcut for `show port`
+portS :: String
+portS = show port
+
+-- | Return a readable stream that emits the first string, then the second
+-- | string, with a delay in between given by the third argument
+foreign import stagger :: String -> String -> Int -> Stream.Readable ()
```

Comment:
Coming originally from Elm, I cringe on examples (or any code) that require FFI. Would it make this example harder to comprehend or its point unclearer to e.g. read a file or run an external process to get the streaming response body?

Reply:
So, I totally 100% agree with you, and was really hesitant to do this.  The reason I decided on this instead of reading a file were:

1) Creating a custom stream is something that `purescript-node-streams` _should_ probably support.  I'm waiting for a response to https://github.com/purescript-node/purescript-node-streams/issues/19; if they accept a PR to add custom streams to the API (since it's part of the Node API, I expect that they will) then we'll be able to get rid of the FFI code and go full native.

2) I didn't want to use files as the example here, because once the file helpers are in place we will want folks to bias towards those over using the `Stream` API.  Having an example that directly contradicts that will be confusing.

Those points said, I think a nice compromise could be using an external process.  I'll definitely look into making that change.